src/test/java/com/puppycrawl/tools/checkstyle/DetailAstImplTest.java

```diff
@@ -641,6 +641,15 @@ public void testToString() {
             .isEqualTo("text[0x0]");
     }
 
+    /**
+     * Returns list of files from a given directory.
+     *
+     * @param dir directory to add files from
+     * @return list of files
+     * @noinspection ParameterNameDiffersFromOverriddenParameter
+     * @noinspectionreason ParameterNameDiffersFromOverriddenParameter -
```

Comment:
Inspections thinks that lambda parameter should have the same name, which would not compile